assets/css/style.css

```diff
@@ -0,0 +1,234 @@
+body {
+  margin: 0px;
+  padding: 0px;
+}
+
+.container {
+  width: 1440px;
```

Comment:
I guess that you wanted to have max-width. With this width it looks way to big for my screen, please fix it. My screen size is 1280px.